dev/kube/kube-ready-state-check.sh

```diff
@@ -41,22 +44,23 @@ function has_command() {
 }
 
 function green() {
-    printf "\033[32m%b\033[0m\n" "$1"
+    printf '\e[32m%b\e[0m' "$1"
 }
 
 function red() {
-    printf "\033[31m%b\033[0m\n" "$1"
+    printf '\e[31m%b\e[0m' "$1"
 }
 
 function verified() {
-    green "Verified: $1"
+    printf 'Verified: %s\n' "$(green "${1}")"
 }
 
 function trouble() {
-    red "Configuration problem detected: $1"
+    printf 'Configuration problem detected: %s\n' "$(red "${1}")"
 }
 
 function status() {
+    # shellcheck disable=SC2181
```

Comment:
We should have a comment on why we're disabling the shellcheck. (But, as mentioned offline, I'm for disabling that here.)

Reply:
Fixed with commit 4a9ca8da6052eb1654d29584c17c897af307b310